os/board/artik05x/src/artik05x_boot.c

```diff
@@ -142,6 +142,19 @@ static void board_i2c_initialize(void)
 #endif
 }
 
+/****************************************************************************
+ * Name: board_spi_initialize
+ *
+ * Description:
+ *  Expose board dependent SPIs
+ ****************************************************************************/
+static void board_spi_initialize(void)
+{
+#if defined(CONFIG_SPI) && defined(CONFIG_S5J_SPI)
+	s5j_spi_register(0);
```

Comment:
You decided make dev for spi, then each board should register driver based on port number.
(artik using 4 ports, bcm using 2ports.)

Reply:
there are no initialize code in existing s5j spi. so could you give me the port number or reference? I'll refer it.